web/app/ui/metalakes/MetalakePath.js

```diff
@@ -20,7 +20,6 @@ const Text = styled(Typography)(({ theme }) => ({
 
 const MetalakePath = props => {
   const { routeParams } = props
-
```

Comment:
I think this is an unnecessary change, can you please revert this line?

Reply:
Reverted.